azext_edge/edge/providers/check/base/deployment.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from functools import partial
+from typing import Any, Callable, Dict, List, Optional
+from enum import Enum
+
+from knack.log import get_logger
+from kubernetes.client.exceptions import ApiException
+from rich.padding import Padding
+
+from .check_manager import CheckManager
+from .nodes import check_nodes
+from .resources import enumerate_ops_service_resources
+from ..common import CoreServiceResourceKinds, ResourceOutputDetailLevel
+from ....common import CheckTaskStatus, ListableEnum
+
+from ....providers.edge_api import EdgeResourceApi
+
```

Comment:
wondering can we remove some lines in the import? or are these for grouping?

Reply:
wdym - everything is used here so you need to import it (and it is better to import from the source)

Reply:
I can rearrange it a bit

Reply:
kk got it!